apps/parsley/src/components/LogPane/index.tsx

```diff
@@ -56,6 +58,11 @@ const LogPane: React.FC<LogPaneProps> = ({ rowCount, rowRenderer }) => {
           setPrettyPrint(true);
         }
         performedScroll.current = true;
+        sendEvent({
+          "jump.to.failing.line.enabled": settings.jumpToFailingLineEnabled,
+          name: "Viewed log with sections and jump to failing line",
+          "sections.enabled": settings.sectionsEnabled,
```

Comment:
possibly want to reference settings as a parent object:
```suggestion
          "settings.jump.to.failing.line.enabled": settings.jumpToFailingLineEnabled,
          name: "Viewed log with sections and jump to failing line",
          "settings.sections.enabled": settings.sectionsEnabled,
```

Reply:
I think it's okay to not indicate hierarchy here because the names are unique

Reply:
I actually agree with Minna here. I'm planning on reviewing all of our span attributes in our projects and adding this hierarchy. So that it better falls in line with otel standards and avoids excessive dot notation.

`settings.jump_to_failing_line.enabled`
`settings.sections.enabled`

I'm planning on doing this in a follow up pr but I just wanted to shed some light on my intentions.

Reply:
ah that's a good callout, `jump.to.failing.line` probably creates excessive namespaces that we would never use otherwise (`jump`, `jump.to`, `jump.to.failing`)